openapi.yml

```diff
@@ -352,12 +419,9 @@ components:
           minLength: 1
           maxLength: 5
         scope:
-          type: string
-          enum: ["LOCAL", "NATIONAL"]
+          $ref: '#/components/schemas/OrganizationScope'
         selectedPecIndex:
-          type: integer
-          minimum: -1
-          maximum: 4
+          $ref: '#/components/schemas/ArrayIndexUpTo4'
```

Comment:
If this schema represents the `PecIndexArray` I'd use that. Naming `ArrayIndexUpTo4` mixes implementation and design.

Reply:
Better would be to use a dictionary object like:

```
{
"1": "first@pec.it",
"2": "second@pec.it"
...
}

```

It will be extensible and will support future use cases where 
numbers are replaced with labels.

Consider - like I said above - that AFAIK there's no guarantee that `pec_array[pec_index]` will be constant in time